testdata/scripts/imports.txt

```diff
@@ -118,6 +120,16 @@ var ReflectValueOfVar = ReflectValueOf{ExportedField: "abc"}
 
 var _ = reflect.TypeOf(ReflectValueOfVar)
 
+type ReflectInDefined struct {
+	ExportedField2   int
+
+	unexportedField2 int
+}
+
+var ReflectInDefinedVar = ReflectInDefined{ExportedField2: 9000}
+
+var _ = reflect.TypeOf(ReflectInDefinedVar)
```

Comment:
What if reflection occurs in another package? 
```
./models
|-- models.go // struct definition here
./usage 
|-- usage.go // reflection on struct used here 
```
Or a function is called like json encode/ decode which uses reflection, but is not part of the same package.

Reply:
Hmm your first point is interesting, hadn't thought about that. I'm guessing that detecting if reflection is only used on a type outside of it's parent package will be difficult to properly handle, because if we're garbling the package that used reflection on an imported type, that type would have already been garbled in it's defined package.

As for your second point, I need to make an issue about that. Detecting other stdlib functions that use reflection is outside the scope of this PR.

Reply:
Yeah just tested and using reflection on an imported type fails... but that's outside of this PR I think.

Reply:
Created 2 issues for both of your points.